keystore/filesystem/server_keystore.go

```diff
@@ -239,8 +242,10 @@ func newFilesystemKeyStore(privateKeyFolder, publicKeyFolder string, storage Sto
 			return nil, err
 		}
 	}
+
+	ctx, _ := context.WithTimeout(context.Background(), network.DefaultNetworkTimeout)
```

Comment:
Can you please tell, why there is a timeout in context?

Reply:
Sure. It is a good practice to have a timeout for any network operation. We can ignore it for our SCellEncrytor as we don't do any network calls, but for KMS-related stuff, we should have some timeout checking. In the case of KMS unavailability or some network issue, the request will exit (in 60 secs) with an error TimoutExceeded

Reply:
And context is a commonly used thing for this case in Go. A great practice to pass context as the first param to the functions